core-model/src/main/java/com/sequenceiq/cloudbreak/domain/SecurityGroup.java

```diff
@@ -118,11 +122,15 @@ public void setSecurityRules(Set<SecurityRule> securityRules) {
     }
 
     public String getSecurityGroupId() {
```

Comment:
I suggest to remove this method and handle the issue far away from the entity :) or rename the method to `getFirstSecurityGroupId()`